OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -59,10 +59,14 @@ class TheWorld : public Component {
     void add(Component* comp) {
         // add it the property list of components that owns and serializes them
         updProperty_components().adoptAndAppendValue(comp);
+        addComponent(comp);
```

Comment:
This should not be done here.  'addComponent()` is marking as a subcomponent and should be done in this component's `extendFinalizeFromProperties()` (which it is doing already!) .